src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/VsTestConsoleWrapper.cs

```diff
@@ -237,12 +237,26 @@ public void StartSession()
     public bool StopTestSession(
         TestSessionInfo testSessionInfo,
         ITestSessionEventsHandler eventsHandler)
+    {
+        return StopTestSession(
+            testSessionInfo,
+            options: null,
+            eventsHandler);
+    }
+
+    /// <inheritdoc/>
+    [Obsolete("This API is not final yet and is subject to changes.", false)]
+    public bool StopTestSession(
```

Comment:
Isn't it better to preserve the original signature order? (i.e. keep the event handler arg second and options last).

Reply:
Same answer as above should apply.